Description: lxc-clone: fix handling of lxc.mount entries
 The 'lxc.mount =' entry can have more than one space, or tabs, before the =.
 We only need to disambiguate from 'lxc.mount.entry'.  So just check for a
 space or tab after mount.
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Forwarded: yes

Index: lxc-0.8.0~rc1/src/lxc/lxc-clone.in
===================================================================
--- lxc-0.8.0~rc1.orig/src/lxc/lxc-clone.in	2012-07-20 09:20:08.848993559 -0500
+++ lxc-0.8.0~rc1/src/lxc/lxc-clone.in	2012-07-20 09:37:06.312976331 -0500
@@ -180,7 +180,7 @@
 sed -i '/lxc.utsname/d' $lxc_path/$lxc_new/config
 echo "lxc.utsname = $hostname" >> $lxc_path/$lxc_new/config
 
-grep "lxc.mount =" $lxc_path/$lxc_new/config >/dev/null 2>&1 && { sed -i '/lxc.mount =/d' $lxc_path/$lxc_new/config; echo "lxc.mount = $lxc_path/$lxc_new/fstab" >> $lxc_path/$lxc_new/config; }
+grep "lxc.mount[ \t]" $lxc_path/$lxc_new/config >/dev/null 2>&1 && { sed -i '/lxc.mount[ \t]/d' $lxc_path/$lxc_new/config; echo "lxc.mount = $lxc_path/$lxc_new/fstab" >> $lxc_path/$lxc_new/config; }
 
 if [ -e  $lxc_path/$lxc_orig/fstab ];then
     cp $lxc_path/$lxc_orig/fstab $lxc_path/$lxc_new/fstab
